src/App.vue

```diff
@@ -33,7 +33,8 @@
     return route.meta.hideNavigation !== true && hasActiveUserId.value
   })
 
-  watch(currentTheme, (theme) => themeSelect({ theme }))
+  const root = document.querySelector(':root')
```

Comment:
Sugestão:
Não seria melhor mover esse código para o `theme-select`?
```suggestion
  const root = document.querySelector(':root')
```